### PR TITLE
Some minor improvements here and there

### PR DESCRIPTION
**Pull request by shlyakpavel**
I wrote these being half asleep, please be careful with merging

### CODE REVIEW

File: Libraries/LibWeb/HTML/SyntaxHighlighter/SyntaxHighlighter.cpp [hunk 1]
```diff
@@ -14,10 +14,8 @@
 
 namespace Web::HTML {
 
-bool SyntaxHighlighter::is_identifier(u64 token) const
+bool SyntaxHighlighter::is_identifier(u64) const
```
Comment (gmta) [inline Libraries/LibWeb/HTML/SyntaxHighlighter/SyntaxHighlighter.cpp:17]: Commit title: `true` -> `false`

File: Libraries/LibTLS/Handshake.cpp [hunk 2]
```diff
@@ -257,8 +257,6 @@ ssize_t TLSv12::handle_handshake_payload(ReadonlyBytes vbuffer)
     auto original_length = buffer_length;
```
Comment (gmta) [inline Libraries/LibTLS/Handshake.cpp:257]: Commit message: `Refer to [..]` or `See the [while condition]`

File: Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp [hunk 3]
```diff
@@ -595,7 +595,7 @@ bool is_animatable_property(PropertyID property_id)
 
 bool is_inherited_property(PropertyID property_id)
 {
-    if (property_id >= first_inherited_shorthand_property_id && property_id <= last_inherited_longhand_property_id)
+    if (property_id >= first_inherited_shorthand_property_id && property_id <= last_inherited_shorthand_property_id)
```
Comment (gmta) [inline Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp:598]: Could this error have been exposed with a test? I.e. testing with a property between last_inherited_shorthand_property_id and `first_inherited_longhand_proeprt_id`?

Comment (gmta) [reply Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp:598]: What's the issue?

Comment (shlyakpavel) [reply Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp:598]: I cannot figure out how to properly expose/check is_inherited_property from CSS/HTML/JS

File: AK/QuickSort.h [hunk 4]
```diff
@@ -113,8 +113,7 @@ void single_pivot_quick_sort(Iterator start, Iterator end, LessThan less_than)
             return;
 
         int pivot_point = size / 2;
-        if (pivot_point)
-            swap(*(start + pivot_point), *start);
+        swap(*(start + pivot_point), *start);
```
Comment (alimpfard) [inline AK/QuickSort.h:116]: Please explain why this is redundant (in the commit message)
At minimum it seems to save two extra iterator accesses, so it's not _completely_ redundant.

Comment (alimpfard) [reply AK/QuickSort.h:116]: Fair enough, please put that reasoning in the commit message :)

File: Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp (hunk 3 above)
Comment (shlyakpavel) [reply Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp:598]: That sounds like the best approach so far. Done

**Pull request merged**